src/Cms/PluginAssets.php

```diff
@@ -25,7 +25,7 @@ class PluginAssets
 	public static function clean(string $pluginName): void
```

Comment:
With the `assets` extension, there can be assets where the absolute path does not match the relative path (e.g. `'some-dependency.css' => __DIR__ . '/vendor/some/dependency/styles.css'`). I think we need to tap into `$plugin->assets()` to look up the whether an asset exists.

Reply:
@lukasbestle I pushed a suggestion

Reply:
I like it. Very clean!